tgui/packages/tgui/interfaces/ChemDispenser.js

```diff
@@ -111,7 +111,7 @@ export const ChemDispenser = (props, context) => {
                 lineHeight={1.75}
                 content={recipe.name}
                 onClick={() => act('dispense_recipe', {
-                  recipe: recipe.name,
+                  recipe: recipe.contents,
                 })} />
```

Comment:
Change this back.

Reply:
you are retarded!

Reply:
i was under the assumption that the process_recipe_list proc would directly read from the recipe list variable, rather than directly parse it from the parameter, and it seems this difference varies across PRs.